portfolio/src/main/webapp/index.html

```diff
@@ -9,7 +9,30 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <p>My name is Josiah Caldwell, I am a computer science student at
+           North Carolina A&T State University and this is my second year at Google. 
+           I spent my last summer working with some wonderful people on the Gmail team in Sunnyvale, CA. 
+           I learned a lot from Google last year and i hope to do the same moving forward.
+      </p>
+      <p>I first got into computer science because I was interested in the concept of hacking. 
+          The idea of being able to enter someone else's machine and do with it as you please still fascinates me. 
+          Of course even more important than the infiltration is the systems defense which is I am specializing in 
+          Cybersecurity for my Bachelor's degree.
+      </p>
+      <p>As far as my interest I enjoy playing video games and watching anime. 
+          My favorite video game series is Dishonored but my favorite singular game is Kid Icarus: Uprising.
+          My favorite anime is K, yes just the letter, the artstyle and the soundwork combine to make a masterpiece.
+      </p>
+      <img src="/images/Mark.png" alt="Mark">
+      <h4>This is my portfolio, enjoy!</h4>
+      <h3>Projects</h3>
+      <ul>
```

Comment:
Do you have links you could provide for additional info on any of these? OK if not.